src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerConnection.java

```diff
@@ -532,4 +532,19 @@ CallableStatement prepareCall(String sql, int nType, int nConcur, int nHold,
      * @return flag for using Bulk Copy API for batch insert operations.
      */
     boolean getUseBulkCopyForBatchInsert();
+
+    /**
+     * Returns value of 'enableBulkCopyCache' from Connection String.
+     *
+     * @param enableBulkCopyCache

```

Comment:
I would call the option `cacheBulkCopyMetadata`
```suggestion
     * @param cacheBulkCopyMetadata
```